src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
-body {
+.page {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+
+  scroll-behavior: smooth;
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: center;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+}
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to order CSS properties in a specific sequence. For example, position-related properties should come before display and box-model properties. Please reorder the properties within the `.product-card` block accordingly.